src/lake/Lake/Build/Module.lean

```diff
@@ -1,7 +1,9 @@
 /-
 Copyright (c) 2021 Microsoft Corporation. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Sebastian Ullrich, Mac Malone
+Authors: Sebastian Ullrich, Mac Malone, Siddharth Bhat
+
+Commands to build a Lean module.
```

Comment:
Module descriptions like this should appear in module docstring (e.g., see `Lake.Build.Common`). The general form is:

```lean
/-! # Module Title
<description>
-/
```